app/assets/javascripts/controllers/middleware_deployment/middleware_deployment_controller.js

```diff
@@ -0,0 +1,38 @@
+ManageIQ.angular.app.controller('middlewareAddDeploymentController', MiddlewareDeploymentCtrl);
```

Comment:
Ideally we'd want it to be present, as UI elements, in the deployments section as well, right ? And  effectively when I started that was my idea of it's location, thus, I guess it still belongs to the "deployments" logic part although being present as UI in the server